client/app/cloud/project/compute/infrastructure/list/cloud-project-compute-infrastructure-list.controller.js

```diff
@@ -41,16 +60,38 @@ class CloudProjectComputeInfrastructureListCtrl {
             this.infra = infra;
             this.table.items = _.map(this.infra.vrack.publicCloud.items, instance => {
                 _.set(instance, "volumes", _.get(this.volumes, instance.id, []));
+                instance.ipv4 = instance.getPublicIpv4();
+                instance.ipv6 = instance.getPublicIpv6();
+                instance.macroRegion = this.RegionService.getMacroRegion(instance.region);
+                instance.statusToTranslate = this.getStatusToTranslate(instance);
+                instance.flavorTranslated = this.$translate.instant(`cpci_vm_flavor_category_${instance.flavor.name}`);
+
+                //patch for some translations that have &#160; html entities
+                instance.flavorTranslated = instance.flavorTranslated .replace("&#160;"," ");
+
                 return instance;
             });
         }).catch(err => {
             this.table.items = [];
             this.CloudMessage.error(`${this.$translate.instant("cpci_errors_init_title")} : ${_.get(err, "data.message", "")}`);
+            this.$log.error(err);
```

Comment:
I'm not sure this is useful.

Reply:
Yes we should log to output error somewhere for us else only the user see it in production. Sentry ideally. I will replace with $q.reject(), it's more appropriate

Reply:
Sad... why errorOnUnhandledRejections is disabled?!
No logging, no sentry.. IN PRODUCTION
https://github.com/ovh-ux/ovh-manager-cloud/blob/master/client/components/q/q.config.js